src/test/mir-opt/storage_ranges.rs

```diff
@@ -22,12 +22,12 @@ fn main() {
 // START rustc.node4.PreTrans.after.mir
 //     bb0: {
 //         nop;                             // scope 0 at storage_ranges.rs:14:9: 14:10
-//         var0 = const 0i32;               // scope 0 at storage_ranges.rs:14:13: 14:14
+//         nop;                             // scope 0 at storage_ranges.rs:14:13: 14:14
```

Comment:
This test should probably use the MIR before any optimization passes, just to check the way storage ranges are generated.